app/views/hq/sponsors/_form.html.haml

```diff
@@ -0,0 +1,105 @@
+= form_for [:hq, @sponsor] do |f|
+  = render 'shared/errors', object: f.object
+
+  .form-group
+    = f.label :name
+    = f.text_field :name, class: "form-control"
+  .form-group
+    = f.label :status_id
+    = f.collection_select :status_id, @statuses, :id, :name, {}, 
+          class: "form-control"
+  .form-group
+    = f.label :gender
+    = f.select :gender, Settings.lookup.gender, {}, class: "form-control"
+  .form-group
+    = f.label :start_date
+    = f.text_field :start_date, class: "form-control date-picker"
+  .form-group
+    = f.label :requested_orphan_count
+    = f.text_field :requested_orphan_count, class: "form-control"
+
+
+  - if f.object.new_record?    
+    .form-group
+      = f.label :sponsor_type
+      = f.collection_select :sponsor_type_id, @sponsor_types, :id, :name, 
+            {}, class: "form-control"
+    .form-group
+      = f.label :organization
+      = f.collection_select :organization_id, @organizations, :id, :name, 
+            {include_blank: true}, class: "form-control"
+    .form-group
+      = f.label :branch
+      = f.collection_select :branch_id, @branches, :id, :name,
+            {include_blank: true}, class: "form-control"
+  - else
+    .form-group
+      = f.label :request_fulfilled
+      = f.check_box :request_fulfilled, 
+          {disabled: true, checked: f.object.request_fulfilled, class: "checkbox"},
+          "#{bool_to_int f.object.request_fulfilled}"
```

Comment:
all you need for this is
```
= f.check_box :request_fulfilled, 
     {disabled: true, class: "checkbox"}
```
the `checked:` property is not needed - form_for takes car of that
the value of 1 or 0 is not needed as that is also take care of